.lychee.yaml

```diff
@@ -0,0 +1,3 @@
+# .lychee.yaml
+exclude:
+  - "https://www.gnu.org/software/libc/"
```

Comment:
Running `mise check links` was failing here as this URL was being ratelimited by the server.